SupportYourLocals.Data/CSVData.cs

```diff
@@ -5,73 +5,95 @@
 using MapControl;
 using Microsoft.VisualBasic.FileIO;
 using SupportYourLocals.ExtensionMethods;
+using System.Security.Cryptography;
 
 namespace SupportYourLocals.Data
 {
-    public class CSVData
+    public class CSVData : IUserStorage
     {
-        private static string filePath = @"./Data.csv";
-        private static int personsID = 1000;
+        private static string filePath = @"./UserData.csv";
 
-        public static void SaveData(String product, Location position)
+        readonly List<UserData> listUserData = new List<UserData>();
```

Comment:
It's more convenient to use a dictionary here, then all the IUserStorage methods will only be a single line.